sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/TypeScriptRules.java

```diff
@@ -19,6 +19,7 @@
  */
 package org.sonar.plugin.typescript;
 
+import java.io.File;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'java.io.File'. ([squid:UselessImportCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-351&issues=AWD6u_KPEFfjxkQcPr_d&open=AWD6u_KPEFfjxkQcPr_d)

[ISSUE_KEY]: AWD6u_KPEFfjxkQcPr_d

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED